pkg/querier/tenantfederation/merge_queryable.go

```diff
@@ -266,13 +323,18 @@ func (m *addLabelsSeriesSet) At() storage.Series {
 // The error that iteration as failed with.
 // When an error occurs, set cannot continue to iterate.
 func (m *addLabelsSeriesSet) Err() error {
-	return m.upstream.Err()
+	return errors.Wrapf(m.upstream.Err(), "error querying %s", m.labels.String())
 }
 
 // A collection of warnings for the whole set.
 // Warnings could be return even iteration has not failed with error.
 func (m *addLabelsSeriesSet) Warnings() storage.Warnings {
-	return m.upstream.Warnings()
+	upstream := m.upstream.Warnings()
+	var warnings = make(storage.Warnings, len(upstream))
```

Comment:
Was this intentional? (actually curious)
```suggestion
	warnings := make(storage.Warnings, len(upstream))
```

Reply:
I think I usually use var when I "initialize" a variable without setting it to some specific value